src/GraphQL.ApiTests/netstandard20+netstandard21/GraphQL.approved.txt

```diff
@@ -1177,7 +1178,6 @@ namespace GraphQL.Execution
     public class ExecutionStrategyRegistration : System.IEquatable<GraphQL.Execution.ExecutionStrategyRegistration>
     {
         public ExecutionStrategyRegistration(GraphQL.Execution.IExecutionStrategy Strategy, GraphQLParser.AST.OperationType Operation) { }
-        protected virtual System.Type EqualityContract { get; }
```

Comment:
Need to undo this change manually -- waiting on https://github.com/PublicApiGenerator/PublicApiGenerator/pull/247